nautobot/dcim/forms.py

```diff
@@ -3633,6 +3625,22 @@ def clean_length_unit(self):
         length_unit = self.cleaned_data.get("length_unit", None)
         return length_unit if length_unit is not None else ""
 
+    def add_error(self, field, error):
```

Comment:
Looks like a clever fix! Can you add some test coverage -- I think adding a function in nautobot.dcim.tests.test_views.CableTestCase might be the appropriate location?